common/oauthTokenManager.go

```diff
@@ -152,7 +152,7 @@ func (uotm *UserOAuthTokenManager) validateAndPersistLogin(oAuthTokenInfo *OAuth
 	}
 	uotm.stashedInfo = oAuthTokenInfo
 
-	if persist && err == nil {
+	if oAuthTokenInfo.Persist && err == nil {
```

Comment:
There is no case where err is not nil here, then why this check?

Reply:
That check exist of err is not nil exist just above when we are getting the token.